clang/www/cxx_dr_status.html

```diff
@@ -10698,7 +10698,7 @@ <h2 id="cxxdr">C++ defect report implementation status</h2>
     <td><a href="https://cplusplus.github.io/CWG/issues/1815.html">1815</a></td>
     <td>CD4</td>
     <td>Lifetime extension in aggregate initialization</td>
-    <td class="none" align="center">No</td>
+    <td class="full" align="center">Clang 19</td>
```

Comment:
```suggestion
    <td class="unreleased" align="center">Clang 19</td>
```
We use a different CSS class here for features that haven't been in an official release yet.

Reply:
Thanks for your tips!